vcluster/_partials/deploy/deploy.mdx

```diff
@@ -0,0 +1,198 @@
+  ]
+}>
+<TabItem value="cli">
+
+<Flow>
+<Step>
+   Install the vCluster CLI.
+
+   <InstallCLIFragment/>
+
+</Step>
+<Step>
+   Deploy vCluster.
+
+   ```bash
+   vcluster create my-vcluster --namespace team-x --values vcluster.yaml
+   ```
+
+   When the installation finishes, you are automatically connected to the virtual cluster. You Kubernetes context is updated to point to your new virtual cluster. You can run local `kubectl` commands for the new virtual cluster.
+</Step>
+</Flow>
+</TabItem>
+
+<TabItem value="helm">
+
+   <Flow>
+   <Step>
+
+      Create the "team-x" namespace.
+
+      ```bash
+      kubectl create namespace team-x
+      ```
+
+   </Step>
+   <Step> 
+
+   Deploy vCluster using `helm upgrade`, which can be used to deploy a new vCluster or re-deploy an existing vCluster with new `vcluster.yaml` values. 
+
+   ```bash
+   helm upgrade --install my-vcluster vcluster \
+   --values vcluster.yaml \
+   --repo https://charts.loft.sh \
+   --namespace team-x \
+   --repository-config='' \
+   ```
+
+   </Step>
+   </Flow>
+
+</TabItem>
+
+<TabItem value="terraform">
+
+<Flow>
+<Step>
+
+   Create a `main.tf` file.
+
+    <CodeBlock language="hcl">{TerraformDeploy}</CodeBlock>
+
+</Step>
+<Step>
+
+   Install the required [Helm provider](https://registry.terraform.io/providers/hashicorp/helm/latest).
+
+   ```bash
+   terraform init
+   ```
+
+</Step>
+<Step>
+
+   Generate a plan.
+
+    ```bash
+    terraform plan
+    ```
+
+   Verify that the provider can access your cluster and that the proposed changes are correct.
+
+</Step>
+<Step>
+   Deploy vCluster.
+
+   ```bash
+   terraform apply
+   ```
+</Step>
+</Flow>
+</TabItem>
+<TabItem value="argo">
+
+To deploy vCluster using ArgoCD, you need the following files:
+
+- `vcluster.yaml` for your configuration options of your vCluster.
+- `my-vcluster-app.yaml` for your ArgoCD `Application` definition.
+
+<Flow>
+<Step>
+
+   Create the ArgoCD `Application` file `my-vcluster-app.yaml`, which refers to the vCluster Helm chart. 
+
+    ```yaml
+    apiVersion: argoproj.io/v1alpha1
+    kind: Application
+    metadata:
+      name: my-vcluster
+      namespace: argocd
+    spec:
+      project: default
+      source:
+        chart: vcluster
+        repoURL: https://charts.loft.sh
+        helm:
+          releaseName: my-vcluster
+          valueFiles:
+            - vcluster.yaml
+      destination:
+        server: https://kubernetes.default.svc
+        namespace: team-x
+    ```
+
+</Step>
+<Step>
+
+   Commit and push these files to your configured ArgoCD repository and synchronize it with your configured cluster.
+
+</Step>
+</Flow>
+
+</TabItem>
+<TabItem value="cluster-api">
+
+<Flow>
+<Step>
+
+   [Install](https://cluster-api.sigs.k8s.io/user/quick-start.html#install-clusterctl) the `clusterctl` CLI.
+
+</Step>
+<Step>
+
+   Install the vCluster provider.
+
+   ```bash
+   clusterctl init --infrastructure vcluster
```

Comment:
```suggestion
   clusterctl init --infrastructure vcluster:v0.2.0-alpha.1
```
Otherwise the old stable release 0.1.3 will be used, which won't work with the current vcluster.yaml format.